EmotionRecogSpeechGUI/app/src/main/java/project/ece496/emotionrecogspeechgui/Record.java

```diff
@@ -0,0 +1,249 @@
+package project.ece496.emotionrecogspeechgui;
+
+
+import android.media.MediaPlayer;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.support.v7.widget.AppCompatButton;
+import android.view.View;
+import android.view.ViewGroup;
+import android.util.Log;
+import java.io.IOException;
+import java.lang.String;
+import project.ece496.emotionrecogspeechgui.RecorderActivity;
+import project.ece496.emotionrecogspeechgui.AudioRecorder;
+
+
+/**
+ * A fragment with a Google +1 button.
+ * Activities that contain this fragment must implement the
+ * {@link Record.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link Record#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class Record extends Fragment {
+    private static final String LOG_TAG = "AudioRecordTest";
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+    // The request code must be 0 or greater.
+    private static final int PLUS_ONE_REQUEST_CODE = 0;
+    // The URL to +1.  Must be a valid URL.
+    private final String PLUS_ONE_URL = "http://developer.android.com";
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+    private AppCompatButton mRecordButton;
+    private AppCompatButton mPlayButton;
+    private AudioRecorder mRecorder = null;
+    private MediaPlayer mPlayer = null;
+    private static String mFileName = null;
+    boolean mStartPlaying = true;
+    boolean mStartRecording = true;
+    private MainActivity main;
+
+    //private OnFragmentInteractionListener mListener;
+
+    public Record() {
```

Comment:
Functionality implementation looks pretty good!!! 
Would suggest to remove all the commented code cuz too much of them makes reading hard with huge scrolls Dx